admin/base/settings/defaults.py

```diff
@@ -23,9 +25,30 @@
     '.osf.io'
 ]
 
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+    },
+    {
+        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+        'OPTIONS': {
+            'min_length': 5,
```

Comment:
This is not having the desired effect that I can see. I am able to reproduce @Ghalko's result at https://github.com/CenterForOpenScience/osf.io/pull/4992#issuecomment-180451300.